superset/assets/src/dashboard/components/menu/MarkdownModeDropdown.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { t } from '../../../locales';
+
+import PopoverDropdown from './PopoverDropdown';
+
+const propTypes = {
+  id: PropTypes.string.isRequired,
+  value: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+export default class MarkdownModeDropdown extends React.PureComponent {
+  render() {
+    const { id, value, onChange } = this.props;
+    const options = [
```

Comment:
can you define the options outside of the render so they aren't created every update?